docs/content/scripts/speedcurve.md

```
@@ -0,0 +1,108 @@
+  spaMode: true,
+  label: to => to.meta.title as string ?? to.path,
+})
+```
+
+Set `label: false` to disable labeling entirely. Pass a plain string to set a static label (only meaningful without `spaMode`, since the router hook overwrites it on every navigation).
+
+## CSP
+
+Add these directives to your Content Security Policy:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)